docs/spec/callables.rst

```diff
@@ -522,31 +520,31 @@ and overloads. They can be defined as protocols with a ``__call__`` member::
       ...
 
   comb: Combiner = good_cb  # OK
-  comb = bad_cb  # Error! Argument 2 has incompatible type because of
+  comb = bad_cb  # Error! Argument 2 is not assignable because of
                  # different parameter name and kind in the callback
 
 Callback protocols and ``Callable[...]`` types can generally be used
 interchangeably.
 
 
-Subtyping rules for callables
------------------------------
+Assignability rules for callables
+---------------------------------
 
-A callable type ``A`` is a subtype of callable type ``B`` if the return type
-of ``A`` is a subtype of the return type of ``B`` and the input signature
-of ``A`` accepts all possible combinations of arguments that the input
-signature of ``B`` accepts. All of the specific subtyping rules described below
-derive from this general rule.
+A callable type ``B`` is :term:`assignable` to callable type ``A`` if the
+return type of ``B`` is assignable to the return type of ``A`` and the input
+signature of ``B`` accepts all possible combinations of arguments that the
+input signature of ``A`` accepts. All of the specific assignability rules
```

Comment:
Not sure if this is useful or not, but the term I like to use when talking about "input signatures" or "parameter specifications" is "strictly more lenient" -- a callable type ``B`` can only be assignable to a callable type ``A`` if `B`'s signature is _strictly more lenient_ than `A`'s. (I.e., it accepts all possible combinations of arguments that `A` accepts, and might also accept some more combinations on top of that.)

Reply:
In the term "strictly more lenient", I find the word "strictly" to be unclear.
Without context I would understand it to imply "not equally lenient" because it makes me think of the difference between "subset" and "strict subset" in set theory.
But that doesn't seem to be how you're using the term here, because 2 callables with exactly the same signature should be assignable to each other, right?

Reply:
Yeah that's a great point, I'm probably not using the term "strictly" very formally there

Reply:
I do like the "more lenient" phrasing as an aid to intuition for callables, but I think introducing and using that term in this doc is out of scope for this PR.